core/templates/pages/library-page/library-page.component.ts

```diff
@@ -110,7 +116,11 @@ export class LibraryPageComponent {
       return;
     }
 
-    let windowWidth = $(window).width() * 0.85;
+    // The number 0 here is just to make sure that the type of width is number,
+    // it is never assigned as the width will never be undefined.
+    let width = $(window).width() || 0;
```

Comment:
While you are here can you remove the usage of `$` and replace it with Angular or pure JS code?

Reply:
ditto below